opencensus/trace/exporters/stackdriver_exporter.py

```diff
@@ -14,12 +14,20 @@
 
 import os
 
+from opencensus.trace.attributes import Attributes
+from opencensus.trace import attributes_helper
 from opencensus.trace.exporters import base
 from opencensus.trace.exporters.transports import sync
 
 from google.cloud.trace.client import Client
 
 
+# OpenCensus Version
+VERSION = '0.1.1'
```

Comment:
Maybe add a note in the release procedure to update this.

Reply:
OK, will add it.